cmd/ocm-backplane/healthCheck/health_check.go

```diff
@@ -0,0 +1,72 @@
+package healthcheck
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	checkVPN   bool
+	checkProxy bool
+)
+
+// HealthCheckCmd is the command for performing health checks
+var HealthCheckCmd = &cobra.Command{
+	Use:   "health-check",
+	Short: "Check VPN and Proxy connectivity",
+	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
It's good to move this as separate function to keep the consistency as other backplane subcommand